### PR TITLE
Fixing darkmode bug

### LINKED ISSUE #25: Styling Issue: Dark Mode Text Visibility
I hope this message finds you well. 
I have encountered a styling issue in the application that affects text visibility in dark mode. Specifically, when dark mode is enabled, the text color remains black, making it challenging for users to read. 

Screenshot 2024-01-29 230716

### PR DESCRIPTION
**Pull request by ichigo-k**
I fixed the bug concerning the dark mode

bug-fixed

fixes #25

### CODE REVIEW

**Comment by blackmann** [comment]
@ichigo-k thanks

Some suggestion:

I think it's easy to miss adding `dark:text-white` wherever possible. So to make things easier, could you try adding

```css
:root {
  @apply dark:text-white;
  ...
}
```

to `style.css`

Then you can remove all additions of `dark:text-white` wherever you added it

**Comment by ichigo-k** [comment]
Okay thanks for the advice

**Review by ichigo-k — COMMENTED** [review]
Fixed dark mode bug issues

File: client/app/style.css [hunk 1]
```diff
@@ -8,6 +8,8 @@
   font-family: 'Inter', system-ui, sans-serif;
   color-scheme: light dark;
   font-size: 14px;
+
+  @apply dark:text-white;
```
Comment (blackmann) [inline client/app/style.css]: There's already an `@apply ...` line in this selector. You can add this at the end; ie,

```css
:root {
  @apply bg-zinc-50 dark:bg-neutral-900 dark:text-white;
  ...
}
```

**Review by blackmann — CHANGES_REQUESTED** [review]

**Comment by ichigo-k** [comment]
Screenshot 2024-01-30 074936

I recently noticed that when I applied the dark:text-white utility to the :root element in the CSS, expecting it to affect the entire work. However, the modal does not seem to be impacted as anticipated.

**Comment by blackmann** [comment]
Try adding another selector/style

```css
dialog {
 @apply dark:text-white;
}
```

I'm guessing this would work. 

---
It's faster to chat on my discord. Can we text there?

File: client/app/components/lesson-form.tsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ function LessonForm({
 
   return (
     <form onSubmit={handleSubmit(addLesson)}>
-      <header className="font-bold text-lg mb-2">Add Lesson</header>
+      <header className="font-bold text-lg mb-2zz">Add Lesson</header>
```
Comment (blackmann) [inline client/app/components/lesson-form.tsx:106]: This zz should go
```suggestion
      <header className="font-bold text-lg mb-2">Add Lesson</header>
```

File: client/app/components/pending-ui.tsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ function PendingUI() {
 
   return (
     <div
-      className="loader sticky left-0 top-0 h-1 -mb-1 bg-blue-500 transition-[width] duration-200 transition-ease-in"
+      className="loader sticky left-0 top-0 h-1 -mb-1 bg-blue-500 transition-[width] duration-200 transition-ease-inzz"
```
Comment (blackmann) [inline client/app/components/pending-ui.tsx:39]: This zz as well

```suggestion
      className="loader sticky left-0 top-0 h-1 -mb-1 bg-blue-500 transition-[width] duration-200 transition-ease-in"
```

File: client/app/routes/timetable_.$year.$programme.$level.$sem.$day.tsx [hunk 4]
```diff
@@ -115,7 +115,9 @@ export default function TimeTable() {
               <div className="size-4">
                 <div className="i-lucide-list-plus opacity-50" />
               </div>{" "}
+              <span className="text-blue-400">
```
Comment (blackmann) [inline client/app/routes/timetable_.$year.$programme.$level.$sem.$day.tsx:118]: Any reason for this?

Comment (ichigo-k) [reply client/app/routes/timetable_.$year.$programme.$level.$sem.$day.tsx:118]: No please it must have been a typo

Comment (ichigo-k) [reply client/app/routes/timetable_.$year.$programme.$level.$sem.$day.tsx:118]: Dw I am going to close my pull request seems like I am causing too much problems , thanks for your time tho

**Comment by ichigo-k** [comment]
btw what is your discord ???

**Pull request closed without merging**